great_ape_safe/ape_api/ebtc.py

```diff
@@ -70,12 +70,26 @@ def __init__(self, safe):
     def schedule_timelock(
         self, timelock, target, value, data, predecessor, salt, delay
     ):
+        """
+        @dev Schedules a timelock transaction.
+        @param timelock The timelock contract to execute the transaction on.
+        @param target The target of the timelock transaction.
+        @param value The ETH value of the timelock transaction.
+        @param data The data of the timelock transaction (encoding of function signature and parameters).
+        @param predecessor The predecessing transacction of the timelock transaction. Matters when cancelling batched transactions.
```

Comment:
i think that not only matters for "cancelling batched txs", but perhaps it is not it meant purpose afaik

sometimes when posting a tx, we will like to highlighted dependancy between them, which i think its the utility of this variable, no?

meaning before predecessor did not get fully exec, do not allow exec the one im queuing, lmk if im missing something

if the case is like that in which particular cases do you think we may use this @param?

Reply:
i think the commentary for this @param may still be worth to revisit on its description to avoid confusion, since still i think that may not be only a **Matters when cancelling batched transactions**, but also in the scenario that one tx its dependant of another one completion

Reply:
Seems like I had actually corrected all instanced. Current comment;
```
@param predecessor The predecessing transacction of the timelock transaction. Matters when there is a dependency between operations.
```

Reference: https://docs.openzeppelin.com/contracts/4.x/api/governance#timelock-terminology